### DISCUSSION TITLE
Isotope plugin

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [bootscore/bootscore#484] [Q&A] [answered]
Hello Basti,

as i purchased your plugin time ago and today started to use it. 

Little favour please, if not asking 2 much ofcourse: one little video (even really simple one) on how to use it because i have tryed to follow up your example but it's pretty unclear for me.

Thank you so much, as always!

**Comment by crftwrk** [comment]
Hi Richard,

what's not clear enough? Where can I help?

**Reply by ritchiemarkov** [reply]
![image](https://github.com/bootscore/bootscore/assets/56654610/345dc16e-6543-453b-9c97-eac9c2defd22)
![image](https://github.com/bootscore/bootscore/assets/56654610/4c607ae3-c5a5-4b6e-a812-070c74c1a9c3)
I dont understand Tax= wha stands for. How can i display this?

**Reply by ritchiemarkov** [reply]
I need an example please, Animals (cats dogs birds). 
And how can i show some particular "category of posts" I mean i have "maps posts" and i wanna show every map.

**Reply by crftwrk** [reply, marked as answer]
As described here https://bootscore.me/documentation/plugin/bs-isotope/#Usage, all "categories" you want to show must be wrapped in a parent caretory. Your categories are correct. `tax=""` is the taxonomy/category, in this case `isotope_category`. The only thing what you need to do is to add the ID of the parent taxonomy. Means, the ID of the category `Animals` into `cat_parent=""` category. For example `123`.

Shortcode should look like this. Change `cat_parent` ID of course.

`[bs-isotope-masonry type="isotope" tax="isotope_category" cat_parent="123"]`

**Reply by ritchiemarkov** [reply]
Ty so much Basti. <3

**Reply by crftwrk** [reply]
You‘re welcome 👍
